model/pipeline/src/main/proto/beam_runner_api.proto

```diff
@@ -296,6 +296,24 @@ message StandardPTransforms {
     //
     // Payload: WindowingStrategy#window_fn FunctionSpec
     MERGE_WINDOWS = 7 [(beam_urn) = "beam:transform:merge_windows:v1"];
+
+    // A transform that translates a given element to its human-readable
+    // representation.
+    // 
+    // Input: KV<nonce, element>
+    // Output: KV<nonce, string>
+    // 
+    // For each given element, the implementation should return the
+    // human-readable representation. When possible, the implementation should
+    // call a user-overridable method on the type. For example, Java should
+    // call `toString()`, Python should call `str()`, Golang should call
+    // `String()`. Users can override `toString`, `__str__`, and `String` for
+    // each language, respectively. The nonce is used by a runner to associate
```

Comment:
I'd omit the detail about what users can override, this is inherent in the previous sentence.